server/tidb_test.go

```diff
@@ -485,7 +512,10 @@ func registerTLSConfig(configName string, caCertPath string, clientCertPath stri
 		ServerName:         serverName,
 		InsecureSkipVerify: !verifyServer,
 	}
-	mysql.RegisterTLSConfig(configName, tlsConfig)
+	err = mysql.RegisterTLSConfig(configName, tlsConfig)
+	if err != nil {
+		return mysql.RegisterTLSConfig(configName, tlsConfig)
+	}
 	return nil
```

Comment:
sorry, I do not make it clear.
```suggestion
	return mysql.RegisterTLSConfig(configName, tlsConfig)
```